include/Isd.h

```diff
@@ -59,19 +60,17 @@ namespace ale {
     double starting_ephemeris_time;
     double center_ephemeris_time;
 
-    double t0_ephemeris_time;
-    double dt_ephemeris_time;
-
-    double t0_quaternions;
-    double dt_quaternions;
-
     json naif_keywords;
+    
+    interpolation interpMethod;
+    
+    Rotation const_rotation; 
 
-    //Positions sensor_pos;
-    //Positions sun_pos;
+    States inst_pos;
+    States sun_pos;
 
-    //Rotation sensor_orientation;
-    //Rotation body_orientaion;
+    Orientations inst_pointing;
+    Orientations body_rotation;
```

Comment:
I think the easiest way to get the constant rotations in is to just add members for them and if they are present in the ISD set them to that, but otherwise set them to the identity rotation.